package.json

```diff
@@ -52,7 +52,7 @@
   "main": "app.js",
   "private": true,
   "scripts": {
-    "start:dev": "vite",
+    "start:dev": "vite --host",
```

Comment:
Is this for Cypress to access it properly?

Reply:
@seriouslysean it lets you access the app on your local wifi network, which was default in vue cli but off by default in vite. I believe it was actually necessary for cypress iirc it uses your local ip to navigate to the tests for some reason. Either way it's nice for testing on mobile etc